astropy/coordinates/tests/test_representation.py

```diff
@@ -1046,6 +1049,16 @@ class LogDRepresentation(BaseRepresentation):
                                         ('lat', Latitude),
                                         ('logr', u.Dex)])
 
+    # now try passing in a differential
+    diff = CartesianDifferential(d_x=1 * u.km/u.s,
```

Comment:
I tried implementing the `LogDDifferential`, but haven't gotten it to work with the dex units - what unit do I pass in for `d_logd`? See `test_representation.py` in incoming commits. I get "`astropy.units.core.UnitTypeError: Quantity instances require <class 'astropy.units.core.UnitBase'> units, not <class 'astropy.units.function.logarithmic.DexUnit'> instances.`" when I try to initialize a `LogDDifferential`.